src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -53,7 +53,11 @@ public partial class CookieCollection : System.Collections.ICollection, System.C
         public object SyncRoot { get { throw null; } }
         public void Add(System.Net.Cookie cookie) { }
         public void Add(System.Net.CookieCollection cookies) { }
-        public System.Collections.IEnumerator GetEnumerator() { throw null; }
+        public void Clear() { }
+        public bool Contains(System.Net.Cookie cookie) { throw null; }
+        public bool Remove(System.Net.Cookie cookie) { throw null; }
+        public System.Collections.Generic.IEnumerator<System.Net.Cookie> GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
```

Comment:
This is a breaking change: you can't take the public `IEnumerator GetEnumerator` method and replace it with `IEnumerator<Cookie> GetEnumerator`.  You will need to make the new generic implementation the explicit implementation and leave the current one public / implicit.